benchmarks-mpp/commonMain/src/benchmarks/immutableList/Add.kt

```diff
@@ -7,10 +7,9 @@ package benchmarks.immutableList
 
 import benchmarks.*
 import kotlinx.collections.immutable.ImmutableList
-import org.openjdk.jmh.annotations.*
-import org.openjdk.jmh.infra.Blackhole
+import kotlinx.benchmark.*
 
-@State(Scope.Thread)
+@State(Scope.Benchmark)
```

Comment:
Benchmark plugin supports only `Scope.Benchmark` state. As we are not running benchmarks in multiple threads, `Scope.Benchmark` and `Scope.Thread` both are fine.

Reply:
👌